### ISSUE TITLE
AutoHitShield

### ISSUE THREAD

**Issue by A-Zalt** [ImpactDevelopment/ImpactClient#1279]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when someone kills me even though I have shield in my offhand

**Describe the solution you'd like**
I want AutoHitShield to automatically block with the shield when a packet of hit is sent from another user.

**Comment by Cerbiac** [comment]
I don't think that would work, b/c you would have already received the packet for damage.

**Comment by VirxEC** [comment]
That's not how it would work, but I love the idea. Maybe how it would work is it would block if there's someone in X range (player can set it) and when the player hits, the shield auto-drops, you hit, then comes back up.
